packages/rich-text/README.md

```diff
@@ -269,11 +269,13 @@ Check if the selection of a Rich Text value is collapsed or not. Collapsed means
 
 _Parameters_
 
--   _value_ `RichTextValue`: The rich text value to check.
+-   _props_ `RichTextValue`: RichTextValue The rich text value to check.
```

Comment:
Was the second 'RichTextValue' on this line intentional?

Reply:
Good question, I think this was autogenerated

Reply:
Yep, I accidentally left it behind in the component's jsdoc :)